configure

```diff
@@ -675,8 +742,8 @@ if test $with_sharedlibs = "yes"; then
       sharedcccompopts="-fPIC"
       mksharedlib="$bytecc -shared"
       bytecclinkopts="$bytecclinkopts -Wl,-E"
-      byteccrpath="-Wl,-rpath,"
-      mksharedlibrpath="-Wl,-rpath,"
+      byteccrpath="-Wl,-rpath -Xlinker "
+      mksharedlibrpath="-Wl,-rpath -Xlinker "
```

Comment:
What does this change have to do with installing the native compilers by default? Was it tested on Linux, FreeBSD, OpenBSD, NetBSD, and Haiku?